### PR TITLE
e2fsprogs - enable shared libs too

### PR DESCRIPTION
**Pull request by tseemann**
This patch just adds `--enable-elf-shlibs` to the command line so that `.so` libs are built as well as `.a` static ones.  The `libcom_err.so` is needed for `table2asn_GFF` being built for `brewsci/bio`.

- [x] Have you followed the [guidelines for contributing](https://github.com/Linuxbrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Linuxbrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/e2fsprogs.rb [hunk 1]
```diff
@@ -19,7 +20,10 @@ class E2fsprogs < Formula
   depends_on "gettext"
 
   def install
-    system "./configure", "--prefix=#{prefix}", "--disable-e2initrd-helper"
+    system "./configure",
+           "--prefix=#{prefix}",
+           "--disable-e2initrd-helper",
+           "--enable-elf-shlibs"
```
Comment (iMichka) [inline Formula/e2fsprogs.rb]: Please write this like this:
`*("--enable-elf-shlibs" unless OS.mac?)`

File: Formula/e2fsprogs.rb [hunk 2]
```diff
@@ -3,6 +3,7 @@ class E2fsprogs < Formula
   homepage "https://e2fsprogs.sourceforge.io/"
   url "https://downloads.sourceforge.net/project/e2fsprogs/e2fsprogs/v1.43.9/e2fsprogs-1.43.9.tar.gz"
   sha256 "5be0ffc01b9720a3f3ccfc86396016baf1334b98751fefa09e0c63eaffdc3897"
+  revision 1
```
Comment (iMichka) [inline Formula/e2fsprogs.rb]: `revision 1 unless OS.mac?`

**Comment by tseemann** [comment]
```
Error: 1 problem in 1 formula
e2fsprogs:
  * The URL https://e2fsprogs.sourceforge.io/ is not reachable (HTTP status code 503)
```

SourceForge currently down.

**Comment by tseemann** [comment]
It's looking like the sourceforge site is dead still.   I have changed home/url to the Linux kernel home, which is more reliable.

**Comment by tseemann** [comment]
```
Error: 1 problem in 1 formula
e2fsprogs:
  * stable: sha256 changed without the version also changing; please create an issue upstream to rule out malicious circumstances and to find out why the file changed.
==> FAILED
```

This is because I changed the `url` i guess....

**Comment by iMichka** [comment]
@tseemann Please do not forget to rebase this :)

**Comment by tseemann** [comment]
I couldn't merge / rebase so I'll close this and resubmit.

**Pull request closed without merging**